glib/src/subclass/object.rs

```diff
@@ -569,14 +569,9 @@ mod test {
 
         let name_changed_triggered = Arc::new(AtomicBool::new(false));
         let name_changed_clone = name_changed_triggered.clone();
-        obj.connect("name-changed", false, move |args| {
-            let _obj = args[0].get::<Object>().expect("Failed to get args[0]");
-            let name = args[1].get::<&str>().expect("Failed to get args[1]");
-
+        obj.connect_static("name-changed", false, move |_obj: Object, name: String| {
```

Comment:
I.e. the goal is it to make this compile:

```suggestion
        obj.connect_static("name-changed", false, move |_obj: Object, name: &str| {
```